### DISCUSSION TITLE
Uploading data to postgis

### DISCUSSION THREAD

**Discussion by dahlalex** [opengeos/leafmap#310] [Q&A] [answered]
Hi @giswqs, thank you so much for this effort!

I wonder if it's possible to save the drawings on the map to postgis?

regards,

**Comment by giswqs** [comment, marked as answer]
You can use `m.save_draw_features('roi.geojson')` to save drawings to a GeoJSON, then use GeoPandas to read it as a GeoDataFrame and save it to PostGIS 

https://geopandas.org/en/stable/docs/reference/api/geopandas.GeoDataFrame.to_postgis.html
